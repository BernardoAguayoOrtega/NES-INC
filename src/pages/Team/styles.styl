.team__grid
  width: 1200px
  margin: 0 auto

@media screen and (max-width: 1023px)
  .team__grid
    width: 98%
@media screen and (min-width: 1024px)
  .team__grid
    width: 98%
@media screen and (min-width: 1200px)
  .team__grid
    width: 1200px