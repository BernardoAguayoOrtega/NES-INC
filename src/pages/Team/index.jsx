// import react
import React from 'react';
// import Hero
import Hero from '../../components/Hero/index.jsx';
// import masonry
import Masonry, { ResponsiveMasonry } from 'react-responsive-masonry';
// import styles
import './styles.styl';



// create and export Team page
const Team = () => {

  return (
    <div className="team__grid">
      <ResponsiveMasonry columnsCountBreakPoints={{ 480: 1, 700: 2 }}>
        <Masonry gutter="20px">
          <a href="https://github.com/BernardoAguayoOrtega">
            <Hero
              type="small"
              caption="Bernardo Aguayo"
              background="https://images.unsplash.com/photo-1593085512500-5d55148d6f0d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/fleandro22">
            <Hero
              type="small"
              caption="Franklin Leandro"
              background="https://images.unsplash.com/photo-1602620502036-e52519d58d92?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/johan-avila">
            <Hero
              type="small"
              caption="Johan Avila"
              background="https://images.unsplash.com/photo-1594007759138-855170ec8dc0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/alejandro1030">
            <Hero
              type="small"
              caption="Alejandro Gonzales"
              background="https://images.unsplash.com/photo-1597422009092-e0f9850b5b56?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/JulesCubs">
            <Hero
              type="small"
              caption="Julian Cubillos"
              background="https://images.unsplash.com/photo-1564412811697-0e26195169ae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/OscarPalominoC">
            <Hero
              type="small"
              caption="Oscar Eduardo"
              background="https://images.unsplash.com/photo-1589539148230-0a5b707cbd7d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a href="https://github.com/ElXavs">
            <Hero
              type="small"
              caption="Xavier Garcia"
              background="https://images.unsplash.com/photo-1590341328520-63256eb32bc3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
          <a>
            <Hero
              type="small"
              caption="Estefany"
              background="https://images.unsplash.com/photo-1566576912321-d58ddd7a6088?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
            />
          </a>
        </Masonry>
      </ResponsiveMasonry>
    </div>
  );
};

export default Team;
