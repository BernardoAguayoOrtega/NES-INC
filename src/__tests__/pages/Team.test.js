import React from 'react';
// import shallow
import { shallow } from 'enzyme';
// import toJson
import toJson from 'enzyme-to-json';
// import app component
import Team from '../../pages/Team/index.jsx';

describe('<Team />', () => {

  const wrapper = shallow(<Team />);

  it('is render the team page?', () => {
    expect(wrapper.exists()).toBe(true)
  });

  it('are there team grid class?', () => {
    expect(wrapper.find('.team__grid').exists()).toBe(true)
  });

  it('are there refLocation className?', () => {
    expect(wrapper.find('.refLocation').exists()).toBe(true)
  });

  it('are there refLocation has ref?', () => {
    expect(wrapper.find('.refLocation').props().className).toBe('refLocation')
  });

  it('matches the snap shot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
