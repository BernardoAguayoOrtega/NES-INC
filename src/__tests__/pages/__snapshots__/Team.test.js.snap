// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Team /> matches the snap shot 1`] = `
<div
  className="team__grid"
>
  <a
    className="refLocation"
    href="https://github.com/BernardoAguayoOrtega"
  >
    <Hero
      background="https://images.unsplash.com/photo-1593085512500-5d55148d6f0d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Bernardo Aguayo"
      type="small"
    />
  </a>
  <a
    href="https://github.com/fleandro22"
  >
    <Hero
      background="https://images.unsplash.com/photo-1602620502036-e52519d58d92?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Franklin Leandro"
      type="small"
    />
  </a>
  <a
    href="https://github.com/johan-avila"
  >
    <Hero
      background="https://images.unsplash.com/photo-1594007759138-855170ec8dc0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Johan Avila"
      type="small"
    />
  </a>
  <a
    href="https://github.com/alejandro1030"
  >
    <Hero
      background="https://images.unsplash.com/photo-1597422009092-e0f9850b5b56?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Alejandro Gonzales"
      type="small"
    />
  </a>
  <a
    href="https://github.com/JulesCubs"
  >
    <Hero
      background="https://images.unsplash.com/photo-1564412811697-0e26195169ae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Julian Cubillos"
      type="small"
    />
  </a>
  <a
    href="https://github.com/OscarPalominoC"
  >
    <Hero
      background="https://images.unsplash.com/photo-1589539148230-0a5b707cbd7d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Oscar Eduardo"
      type="small"
    />
  </a>
  <a
    href="https://github.com/ElXavs"
  >
    <Hero
      background="https://images.unsplash.com/photo-1590341328520-63256eb32bc3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Xavier Garcia"
      type="small"
    />
  </a>
  <a>
    <Hero
      background="https://images.unsplash.com/photo-1566576912321-d58ddd7a6088?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
      caption="Estefany"
      type="small"
    />
  </a>
</div>
`;
